dist/profile/manifests/kubernetes/resources/accountapp.pp

```diff
@@ -0,0 +1,134 @@
+#   Class: profile::kubernetes::resources::accountapp
+#
+#   This class deploy plugins jenkins website
+#
+#   Parameters:
+#     $image_tag:
+#       Set accountapp image tag.
+#     $election_candidates:
+#       coma separated list of candidates.
+#     $election_close:
+#       date election will close. yyyy/MM/dd
+#     $election_open:
+#       date app will start collecting votes. yyyy/MM/dd
+#     $election_logdir:
+#       collected votes directory.
+#     $jira_username:
+#       jira username
+#     $jira_url:
+#       jira url endpoint
+#     $jira_password:
+#       jira password 
+#     $ldap_manager_dn:
+#       ldap manager dn
+#     $ldap_new_user_base_dn:
+#       ldap new user base dn
+#     $ldap_password:
+#       ldap password
+#     $ldap_url:
+#       ldap endpoint
+#     $recaptcha_private_key:
+#       recaptcha private key 
+#     $recaptcha_public_key:
+#       recaptcha public key
+#     $smtp_server:
+#       smtp server
+#     $smtp_user:
+#       smtp user
+#     $smtp_password:
+#       smtp password
+#     $smtp_auth:
+#       smtp authentication (boolean)
+#     $url:
+#       account app url endpoint
+#
+#   Remark:
+#     `kubectl get service nginx --namespace nginx-ingress` return service public ip
+#     This public ip should be used for DNS record frontend_url -> this IP
+#     In order to create letsencrypt certificates
+#
+# Deploy accountapp resources on kubernetes cluster
+class profile::kubernetes::resources::accountapp (
+    String $election_close = '1970/01/02',
```

Comment:
The accountapp do not support ISO-8601 at the moment but it can be easily changed